com.github.adrienverge.PhotoCollage.yml

```diff
@@ -0,0 +1,24 @@
+app-id: com.github.adrienverge.PhotoCollage
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+rename-icon: photocollage
+rename-appdata-file: photocollage.appdata.xml
+rename-desktop-file: photocollage.desktop
+command: photocollage
+finish-args:
+  - --device=dri # hardware acceleration
+  - --filesystem=home
```

Comment:
home filesystem permissions is broad.

Reply:
It failed to open some files via drag and drop otherwise...

Reply:
it doesn't support portals either. I can't add any file if I remote the filesystem permission.